server/data_access_layer/models/localdb/chart_report.model.js

```diff
@@ -0,0 +1,73 @@
+module.exports = (localdb, Sequelize) => {
+    const ChartReport = localdb.define("chart_reports", {
+        chartReportId: {
+            field: 'chart_report_id',
+            type: Sequelize.UUID,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true
+        },
+        name: {
+            field: 'chart_report_name',
+            type: Sequelize.STRING
```

Comment:
should not allow null values

Reply:
I checked the Sequelize docs and it doesnt explicitly say what value they give allowNull by default. I will add `allowNull: false` just to make sure.